pkg/image/controller/controller_test.go

```diff
@@ -85,6 +87,168 @@ func TestControllerNoDockerRepo(t *testing.T) {
 	}
 }
 
+func TestControllerNoDockerRepo(t *testing.T) {
+	cli, fake := &fakeDockerRegistryClient{}, &client.Fake{}
+	c := ImportController{client: cli, streams: fake, mappings: fake}
+
+	stream := api.ImageStream{
+		ObjectMeta: kapi.ObjectMeta{
+			Name:      "test",
+			Namespace: "other",
+		},
+	}
+	if err := c.Next(&stream); err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if len(stream.Annotations["openshift.io/image.dockerRepositoryCheck"]) == 0 {
```

Comment:
You are using it likewise in a couple of other places in the tests. Update them too